pkg/migrator.go

```diff
@@ -67,7 +87,7 @@ func Migrate(currentConfig map[string]interface{}, vFrom int, vTo *int, mp Migra
 }
 
 func apply(valuesData map[string]interface{}, mTemplate string) (map[string]interface{}, error) {
-	parsedTemplate, err := template.New("migration").Funcs(extraFuncs()).Parse(mTemplate)
+	parsedTemplate, err := template.New("migration").Option("missingkey=zero").Funcs(extraFuncs()).Parse(mTemplate)
```

Comment:
What does this option do?

Reply:
Ah, I can remove that.  It (in theory) stops it putting `<no-value>` in the output if no values are found for a key.  It doesn't appear to work for our use case though, but fortunately we don't actually need it.

Reply:
It's gone now 😄